src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -204,7 +204,7 @@ public static class PermutationFeatureImportanceExtensions
         /// <param name="catalog">The clustering catalog.</param>
         /// <param name="predictionTransformer">The model on which to evaluate feature importance.</param>
         /// <param name="data">The evaluation data set.</param>
-        /// <param name="labelColumnName">Label column name.</param>
+        /// <param name="labelColumnName">Label column name. The column data must be <see cref="System.UInt32"/>.</param>
```

Comment:
Should this be <see cref="KeyDataViewType" />.

Reply:
Yes I agree

---
In reply to: [277186918](https://github.com/dotnet/machinelearning/pull/3455#discussion_r277186918) [](ancestors = 277186918)